packages/webapp/pages/plus/success.tsx

```diff
@@ -44,15 +43,15 @@ const PlusSuccessPage = (): ReactElement => {
           >
             Success! Your payment is complete, you’re all set.
           </Typography>
-          <Link href={webappUrl} passHref>
-            <Button
-              variant={ButtonVariant.Primary}
-              tag="a"
-              size={ButtonSize.Large}
-            >
-              Back to daily.dev
-            </Button>
-          </Link>
+          <Button
+            variant={ButtonVariant.Primary}
+            size={ButtonSize.Large}
+            onClick={() => {
+              window.location.href = webappUrl;
+            }}
```

Comment:
Wouldn't it also work keeping `tag="a"` and setting `href` instead of parent `Link` element?

Reply:
yeah